examples/prometheus/role.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: netobserv-prom
+  namespace: netobserv
+rules:
+  - verbs:
+      - create
```

Comment:
I'm wondering if we really need that `create` here.
Giving another try today

Reply:
so I confirm we need the `create` as is but the behavior could be improved:
https://redhat-internal.slack.com/archives/C0VMT03S5/p1724752880713839?thread_ts=1715862728.898369&cid=C0VMT03S5